.github/.kodiak.toml

```diff
@@ -35,3 +35,6 @@ include_pull_request_author = false # default: false
 
 # remove html comments to auto remove PR templates.
 strip_html_comments = true # default: false
+
+# remove PR template from merge message body
+cut_body_after = "Azure IoTEdge PR checklist:"
```

Comment:
Would you mind adding the space to "IoT Edge", here and in the PR template?

Reply:
Done